ci-operator/config/openshift/oc/openshift-oc-master.yaml

```diff
@@ -101,7 +101,7 @@ rpm_build_commands: |-
   tar -czf "${TMPDIR}/openshift-clients-0.0.1.tar.gz" --exclude='.git' --transform='s|^.|openshift-clients-0.0.1|' --exclude='openshift-clients-0.0.1.tar.gz' ./
   cp ./oc.spec "${RPMBUILD_DIR}/SPECS/"
   cp "${TMPDIR}/openshift-clients-0.0.1.tar.gz" "${RPMBUILD_DIR}/SOURCES/"
-  rpmbuild -ba --nodeps --define "_topdir $( pwd )/${RPMBUILD_DIR}" ${RPMBUILD_DIR}/SPECS/oc.spec --define 'os_git_vars ignore'
+  rpmbuild -ba --nodeps --define "_topdir $( pwd )/${RPMBUILD_DIR}" ${RPMBUILD_DIR}/SPECS/oc.spec --define "os_git_vars OS_GIT_VERSION='4.6' OS_GIT_COMMIT='${OPENSHIFT_BUILD_COMMIT::8}' OS_GIT_MAJOR='' OS_GIT_MINOR='' OS_GIT_TREE_STATE=''"
```

Comment:
I'm ok-ish with the rest being set, but for master, I'd prefer not to set it since that will require us changing it on every release and I bet we'll forget about it.

Reply:
If its not set it would pick the latest tag - and show 4.2 :(

I've update master script to have `OS_GIT_VERSION` on top, so that it could be easily spotted. In any case, it would affect OKD (and CI runs) only, so it can be efforlessly resolved

Reply:
I'd rather master be `0.0.0` in that case, not a particular release until it's actually being cut.

Reply:
SGTM, updated